src/bootstrap.php

```diff
@@ -35,7 +35,13 @@
  */
 
 try {
-    $dotenv = new Dotenv\Dotenv(dirname(__FILE__));
+    $reflectDotEnv = new ReflectionMethod('Dotenv\Dotenv', '__construct');
```

Comment:
Will this still work after it has been pearified?

Reply:
This file is not pearified, has no namespace, therefore is handled by composer's autoloader. Should not be an issue.